src/classes/STG_PanelContactRoles_CTRL.cls

```diff
@@ -87,13 +87,20 @@ public with sharing class STG_PanelContactRoles_CTRL extends STG_Panel {
             STG_Panel.stgService.stgHH.npo02__Household_Member_Contact_Role__c = '';
         }
 
-        //add error for blank Primary Contact Role value
+        //add error for blank Default Individual Contact Role value
         if (String.isBlank(STG_Panel.stgService.stgCon.npe01__Opportunity_Contact_Role_Default_role__c)) {
             String defaultRoleLabel = UTIL_Describe.getFieldLabel('npe01__Contacts_And_Orgs_Settings__c', 'npe01__Opportunity_Contact_Role_Default_role__c');
             ApexPages.addMessage(new ApexPages.message(ApexPages.Severity.ERROR, defaultRoleLabel + ' ' + Label.isARequiredField));
             isValid = false;
         }
 
+        // Add error for blank Default Oranizational Contact Role value
+        if (String.isBlank(STG_Panel.stgService.stgCon.Contact_Role_for_Organizational_Opps__c)) {
+            String defaultOrganizationalRoleLabel = UTIL_Describe.getFieldLabel('npe01__Contacts_And_Orgs_Settings__c', 'Contact_Role_for_Organizational_Opps__c');
```

Comment:
I think this is another spot that won't work in a namespaced org because the `"Contact_Role_for_Organizational_Opps__c"` fieldname is hardcoded without a namespace. You can call `UTIL_Namespace.StrTokenNSPrefix()` with the fieldname to add the namespace.

Reply:
@force2b -- Good catch! Thanks for identifying this issue. I've updated the code per your suggestion and it ran successfully in both a namespaced and a regular scratch org.